internal/commands/user.go

```diff
@@ -97,7 +98,7 @@ func getUsersInConversation(
 	for _, id := range *usersIDs {
 		user, err := getSlackUserInfo(ctx, app, id)
 		if err != nil {
-			app.Logger.Error(
+			logWriter.Error(
 				ctx,
 				"command/user.getUsersInConversation",
```

Comment:
Also, add a log.Action("getSlackUserInfo") to identify in what part of the func the error was thrown.